zenoh-protocol/src/link/locator.rs

```diff
@@ -36,13 +38,17 @@ pub const PORT_SEPARATOR: char = ':';
 pub const STR_TCP: &str = "tcp";
 #[cfg(feature = "udp")]
 pub const STR_UDP: &str = "udp";
+#[cfg(all(feature = "unix", target_family = "unix"))]
+pub const STR_UNIX: &str = "unix";
```

Comment:
My proposal for the locator protocol is to use "uds" from "unix domain socket". Other proposals are also accepted.